### ISSUE TITLE
Documentation of meetings as Issue

### ISSUE THREAD

**Issue by AlexanderTheGrape** [softeng-701-group-5/softeng-701-assignment-1#34]
**Description**

As described in the assignment 1 PDF, all meetings that discuss changes to the management of the project must be documented as an issue or pull request. My understanding is that future wiki additions should be made to the wiki through an issue or pull request. This issue can be where this information is documented, in addition to the group 'meetings' discord channel. 
In the case that a new issue should be made every time a meeting is recorded and added to a wiki, this issue can be closed.

"All coordination should occur via issue or pull request comments. In addition, you will meet as a group in the Project Coordination lectures. As a result of these meetings, someone in the
team should document any discussed changes to the management of the project in the
project’s documentation. Any discussions related to specific issues, pull requests, or commits
should be documented as a comment on the issue/pull request/commit. In all cases, be clear
that the comment is recording a discussion that occurred in the group meeting."

**Acceptance Criteria**

- Decision made on where the team meeting info should be kept

**Notes**
---

**Comment by Matteas-Eden** [comment]
I'm afraid to say I slightly disagree with the assignment spec, particularly the comment that
> All coordination should occur via issue or pull request comments

Surely this is misguided - we've been coordinating quite well across Discord and Messenger.
Additionally, do accept my apologies for not agreeing with your interpretation that
> all meetings that discuss changes to the management of the project must be documented as an issue or pull request

The assignment spec you have quoted states that we should
> document any discussed changes to the management of the project in the
project’s documentation

The project's documentation is contained in the wiki, _not_ in issues/pull requests.

In any case, addressing the issue of additions to the wiki through PRs; while I can understand the intention behind the request (it was my first thought too), my research into the matter indicated that it was not all that feasible. Quite simply, **[GitHub doesn't support pull requests for the wiki repository, only the main repository](https://stackoverflow.com/questions/10642928/how-to-pull-request-a-wiki-page-on-github).** As such, I made the decision to edit the wiki directly. The thread I linked does suggest a workaround, but I felt it too inconvenient to put into place.
Going forward, it is perfectly fine by me if issues are created for requesting documentation on the wiki - they just won't be closed by pull requests.

I'm open to discussion on this as there are several ways that we can handle documentation.
